webapi/src/main/java/org/sakaiproject/webapi/controllers/RubricsRestController.java

```diff
@@ -175,6 +175,16 @@ ResponseEntity<EntityModel<CriterionTransferBean>> getDefaultCriterion(@PathVari
             .orElseGet(() -> ResponseEntity.badRequest().build());
     }
 
+    @GetMapping(value = "/sites/{siteId}/rubrics/{rubricId}/criteria/defaultEmpty", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I feel like this should be a PUT. In effect you're putting a new criterion on the rubric. Maybe the url path should be /sites/{siteId}/rubrics/{rubricId}/criteria/empty